docs/source/plugins/06_solver_hooks.rst

```diff
@@ -260,6 +267,12 @@ the implementation of a relative emulsion viscosity correlation.
 
 .. autofunction:: alfasim_sdk._internal.hook_specs.calculate_relative_emulsion_viscosity
 
+.. warning::
+    The emulsion viscosity hook must be implemented as a simple correlation, in which all data is available as
```

Comment:
I think we could avoid mentioning "must be implemented as a simple correlation", since they are not necessarily simple, just have a limited access to the API ;)

```suggestion
    The relative emulsion viscosity hook has a limited API. Therefore the user-implemented correlations will have access only to dispersed field viscosity, continuous field viscosity and volume fraction of the dispersed phase. These variables are passed as arguments of the hook signature.
```

Reply:
Nice suggestion.
Thanks.